contracd/src/providediagnostickeys.cpp

```diff
@@ -226,6 +227,8 @@ void ProvideDiagnosticKeys::run()
     QList<DiagnosisKey> diagnosisKeys[DAYS_TO_STORE];
     bool terminate = false;
 
+    QElapsedTimer buildKeysTimer;
+    buildKeysTimer.start();
```

Comment:
Since the timers are only for debugging purposes, I think they shouldn't run on release builds.
```suggestion
#ifndef NDEBUG
    QElapsedTimer buildKeysTimer;
    buildKeysTimer.start();
#endif
```

Reply:
hmm, i would rather skip the timers then, i dropped the commit with the debug timers from this merge request, see https://github.com/cnlpete/harbour-contrac/commit/d0422ba2f59fad531c9f9fd91e26cb4c137ba3b2